src/main/java/cloud/fogbow/ras/core/plugins/interoperability/aws/compute/v2/AwsV2ComputePlugin.java

```diff
@@ -233,16 +209,14 @@ protected Image getImageById(String imageId, Ec2Client client) throws FogbowExce
 				.imageIds(imageId)
 				.build();
 		
-		DescribeImagesResponse response = doDescribeImagesRequests(request, client);
+		DescribeImagesResponse response = AwsV2CloudUtil.doDescribeImagesRequest(request, client);
```

Comment:
the `updateInstanceAllocation` uses a sync block. Not saying it's wrong but it is surprising we miss other blocks in the other plugins

Reply:
in openStackComputePlugin there is a comment that tries to explain the reason of that..
line 106